ipa-core/src/protocol/basics/partial_reveal.rs

```diff
@@ -1,37 +1,13 @@
 use async_trait::async_trait;
-use futures::future::try_join;
 
 use crate::{
-    error::Error,
-    helpers::{Direction, Role},
-    protocol::{
-        context::{Context, UpgradedMaliciousContext},
-        RecordBinding, RecordId,
-    },
-    secret_sharing::{
-        replicated::{
-            malicious::{AdditiveShare as MaliciousReplicated, ExtendableField},
-            semi_honest::AdditiveShare as Replicated,
-        },
-        SharedValue,
-    },
+    protocol::{context::Context, RecordBinding, RecordId},
+    secret_sharing::{replicated::semi_honest::AdditiveShare as Replicated, SharedValue},
 };
 
-/// Trait for partial reveal protocol to open a shared secret to all helpers except helper `left_out` inside the MPC ring.
 #[async_trait]
 pub trait PartialReveal<C: Context, B: RecordBinding>: Sized {
```

Comment:
Did you mean to delete this entirely, or are you thinking that can wait for a follow-up?

Reply:
oh thanks for catching that - I meant to completely remove it but left some stuff behind. Going to delete the definition as well